woocommerce/woocommerce-android
# Release More Menu Screen

Closes: #5580


### Description


This releases all the work listed in #5580, related to the Milestone 1-2 of the Home Screen project, where the **"More Menu"** screen is added.

### Testing instructions


Every feature here has been tested in its own specific PR (see issue above for list), so for this release, smoke testing should be enough: 

1. Open the "More" menu screen on the main bottom navigation, and test the menu items, Switch Store, and Settings option there,
2. Create a test review in a test site, then open the site and ensure the "More" menu icon and the "Review" menu item show badge icon.

### Screenshot


<img src="https://user-images.githubusercontent.com/266376/154642336-da2149dd-a2fb-4ade-af08-09793a4f1e64.png" width="42%" />
